src/sentry/static/sentry/app/views/organizationDiscover/index.jsx

```diff
@@ -26,19 +25,21 @@ import {
   LoadingContainer,
 } from './styles';
 
-const OrganizationDiscoverContainer = createReactClass({
-  displayName: 'OrganizationDiscoverContainer',
-  mixins: [OrganizationState],
+export default class OrganizationDiscoverContainer extends React.Component {
+  static contextTypes = {
+    organization: SentryTypes.Organization,
```

Comment:
Let's use `withOrganization`

Reply:
What's the benefit of that? Doesn't it require us to export another object just for testing?

Reply:
I'm going to leave this for now since it seems like a stylistic preference and contextTypes looks like it's used extensively throughout the application and withOrganization only in 4 places

Reply:
So we can abstract how we're accessing `organization`, just in case we change how organization is stored/accessed. You won't need to export to test since you can just pass `organization` as a property in tests.